README.md

```diff
@@ -0,0 +1,113 @@
+| shadowflow.flows                                   | java.util.Map<java.lang.String,io.github.rabobank.shadowflow.autoconfiguration.ShadowFlowProperties$ShadowFlowConfig> | {}            | Map of shadow flows configurations.                                                                                                             |
+| shadowflow.encryption                              | io.github.rabobank.shadowflow.autoconfiguration.ShadowFlowProperties$EncryptionProperties                             | n/a           | Encryption options for the shadow flows.                                                                                                        |
+| shadowflow.encryption.cipher                       | io.github.rabobank.shadowflow.autoconfiguration.ShadowFlowProperties$EncryptionProperties$CipherProperties            | n/a           | Cipher properties for encryption.                                                                                                               |
+| shadowflow.encryption.cipher.secret                | java.lang.String                                                                                                      | ""            | The secret for encryption. Should be a 16, 24, or 32-byte string. Could be generated as follows: openssl rand -hex 32                           |
+| shadowflow.encryption.cipher.initialization-vector | java.lang.String                                                                                                      | ""            | The initialization vector for encryption. Should be a 12-byte string. Could be generated as follows: openssl rand -hex 12                       |
+| shadowflow.encryption.public-key                   | java.lang.String                                                                                                      | ""            | Base 64 encoded version of an X509 Public Key. Used in a Cipher with algorithm "RSA/ECB/OAEPWITHSHA-256ANDMGF1PADDING".                         |
+| shadowflow.encryption.noop                         | java.lang.Boolean                                                                                                     | false         | Disables encryption but encodes differences as Base64.                                                                                          |
+| shadowflow.flows[*].percentage                     | java.lang.Integer                                                                                                     | 0             | Percentage of how many calls should be compared in the shadow flow. Should be in the range of 0-100. Zero effectively disables the shadow flow. |
```

Comment:
Think these will be more readable if wrapped in back-ticks. But moreover, I think we can ditch the type, or at least simplify them a lot. For example, nobody cares it's actually a `ShadowFlowConfig`, it's just a map of Shadow Flows. Same about CipherProperties, it's irrelevant: the nested fields are relevant, not the "root".

And you can remove things like `java.lang` here too IMO.

IMO everything that looks like code or a script should be wrapped with back-ticks. e.g `RSA/ECB/OAEPWITHSHA-256ANDMGF1PADDING` and `openssl rand -hex 12`